gnome/garagejam/Portfile

```diff
@@ -75,3 +69,18 @@ if {![variant_isset x11]} {
 if {![variant_isset quartz] && ![variant_isset x11]} {
     error "Either +x11 or +quartz is required"
 }
+
+# Pre-fetch check for Xorg-server
+pre-fetch {
+  if {![file exists ${prefix}/bin/Xorg]} {
+      ui_warn "Xorg-server is required to run GarageJam but is not
+ installed."
+      ui_warn "Install it with: sudo port install xorg-server"
+  }
+}
+# Inform users about manual dbus activation
+notes "
+GarageJam requires D-Bus to be running. If GarageJam does not start, try
+ running:
+    sudo port load dbus
+"
```

Comment:
same here if the "dbus" port is needed, shouldn't the port depend on it? (Perhaps it indirectly does already I haven't checked.)